functions/AFQ_WholebrainTractography.m

```diff
@@ -77,7 +80,15 @@
 %% Track with mrtrix if the right files are there
 if exist('mrtrix','var') && mrtrix == 1
     mrtrixpaths = AFQ_get(params,'mrtrix paths',params.currentsub);
-    [status, results, fg, pathstr] = mrtrix_track(mrtrixpaths, mrtrixpaths.wm, mrtrixpaths.wm, 'prob', opts.nfibers,[],[],1);
+    [status, results, fg, pathstr] = AFQ_mrtrix_track(mrtrixpaths, ...
+                                                      mrtrixpaths.wm, ...
+                                                      mrtrixpaths.wm,...
+                                                      'stream', ...
+                                                      opts.nfibers,...
```

Comment:
Lets add a new option into the afq structure that sets the tractography algorithm used here. We can make SD_Stream the default but allow users a choice